assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -44,7 +45,7 @@ const Overflow = styled.div`
 `;
 
 // @TODO update with actual figure.
-const Figure = styled.div`
+const Figure = styled.video`
```

Comment:
TIL about the video element. This is dope

Reply:
[HTML5 is AWESOME](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5)